crd-generator/api/src/main/java/io/fabric8/crd/generator/AbstractJsonSchema.java

```diff
@@ -114,7 +126,8 @@ protected T internalFrom(TypeDef definition, String... ignore) {
         .emptySet();
     List<String> required = new ArrayList<>();
 
-    for (Property property : Types.projectProperties(definition)) {
+//    for (Property property : Types.projectProperties(definition)) {
```

Comment:
let's remove the comment

Reply:
Yep, I was planning on cleaning these up, looking at Sonar…